docs/content/deployment/dagster-daemon.mdx

```diff
@@ -27,6 +27,8 @@ The following daemons are currently available:
 
 - The _sensor daemon_ is responsible for creating runs from any [sensors](/concepts/partitions-schedules-sensors/sensors) that are turned on. This daemon is always enabled.
 
+- The _run monitoring daemon_ is responsible for handling run worker failures. This daemon can be enabled with the [`run_monitoring`](/deployment/run-monitoring) field in your instance.
```

Comment:
is the term run worker defined here, should we define it here or somewhere that defines it? Something like

maybe "is responsible for monitoring and retrying failed runs"?